start-xvfb-nginx.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
+cleanup() {
+    kill $XVFB_PID $QGIS_PID
```

Comment:
add the 2 lines below reduces uncertainty: 
```bash
sleep 3
kill -KILL $XVFB_PID $QGIS_PID
```

Reply:
I added a comment in the script: timeout is managed by docker via `-t`: https://docs.docker.com/engine/reference/commandline/stop/